frontend/packages/console-dynamic-plugin-sdk/src/app/redux-types.ts

```diff
@@ -18,4 +21,5 @@ export type CoreState = {
 
 export type SDKStoreState = {
```

Comment:
Nit: I'd suggest naming this `SDKReduxState` or `SDKRootState` - AFAIK this is a logical subset of Console `RootState` type, used to manage data related to various plugin SDK utilities.

(We already have `PluginStore` so with Redux this means two different "stores".)

Reply:
A valid point, let us hold this as is for this PR and work to make it in another one. Naming, as we all agree, is hard. This is probably my naming I gave to Jai some weeks ago.

Reply:
OK, agreed :+1: